Cargo.toml

```diff
@@ -44,7 +43,7 @@ serde_json = "^1.0.*"
 slice-of-array = "0.3.*"
 splines = { version = "4.0.*", features = ["serde"] }
 statrs = "0.15.*"
-terminal_size = "0.1.*"
```

Comment:
:o

Reply:
Just bumping version numbers here. Checked that it worked so should be all good.